src/test/java/com/exasol/adapter/dynamodb/documentpath/DocumentPathExpressionTest.java

```diff
@@ -0,0 +1,65 @@
+package com.exasol.adapter.dynamodb.documentpath;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests for {@link DocumentPathExpression}
```

Comment:
That's quite obvious without the comment. You can remove it

Reply:
And also for other tests

Reply:
I like it as you can click on it and directly jump to the tested class

Reply:
You can place the cursor on the class name and press Ctrl +Shift + T
Then you don't need the comments anymore. 
And the same works if you press this combination on the class name itself: it opens the test class.